src/neo4j_genai/generation/prompts.py

```diff
@@ -29,7 +33,7 @@ class PromptTemplate:
     """
 
     DEFAULT_TEMPLATE: str = ""
-    EXPECTED_INPUTS: list[str] = []
+    EXPECTED_INPUTS: list[str] = list()
```

Comment:
For my culture, what's the difference here between `[]` and `list()`?

Reply:
`[]` is a little faster than `list()`, but `list()` is just more explicit. I suppose it's an opinion, but also for `{}` I prefer to explicitly use `dict()` or `set()` for clarity.